dev-db/etcd/files/etcd2.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=etcd
+
+[Service]
+User=etcd
+PermissionsStartOnly=true
+Environment=ETCD_DATA_DIR=/var/lib/etcd
+ExecStart=/usr/bin/etcd
```

Comment:
sorry, i did not mean the data dir, but thats a good point too. i was referencing `newbin "${WORKDIR}/gopath/bin/${PN}" etcd2` in the ebuild. this service file is called 'etcd2.service' so i thought it would call /usr/bin/etcd2 installed by the etcd2 ebuild.